api/schemas.py

```diff
@@ -0,0 +1,15 @@
+from datetime import date
+
+from marshmallow import Schema, fields, ValidationError
+
+
+def validate_date(given_date):
+    if given_date > date.today():
+        raise ValidationError('Enter existing date')
+
+
+class PayloadSchema(Schema):
+    api_key = fields.String(missing='DEMO_KEY')
```

Comment:
Move it 'DEMO_KEY' to config, please

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/0b16b705764a94293e87719fd037611c8fcab3c5